spec/models/draft_spec.rb

```diff
@@ -55,6 +55,13 @@
     Draft.cleanup!
 
     expect(Draft.count).to eq 1
+
+    # should cleanup drafts more than 180 days old
+    SiteSetting.stubs(:delete_drafts_older_than_n_days).returns(180)
```

Comment:
minor but simpler 

SiteSetting.delete_drafts_older_than_n_days = 180

no need to stub here .